link/link.go

```diff
@@ -92,11 +99,40 @@ type RawLinkOptions struct {
 	Flags uint32
 }
 
-// RawLinkInfo contains metadata on a link.
-type RawLinkInfo struct {
+// LinkInfo contains metadata on a link.
+type LinkInfo struct {
 	Type    Type
 	ID      ID
 	Program ebpf.ProgramID
+	extra   interface{}
+}
+
+// RawLinkInfo deprecated: use LinkInfo instead.
```

Comment:
I think the Deprecated has to be on it's own line for linters to do their work.

```suggestion
// RawLinkInfo contains information on a raw link.
//
// Deprecated: use LinkInfo instead.
```

Reply:
Updated.